tyr-core/src/main/resources/format-example.yaml

```diff
@@ -25,6 +25,5 @@ format:
     AddUserCommand: "ok\\s+to\\s+test$"
     RetestCommand: "retest\\s+this\\s+please$"
     RetestFailedCommand: "retest\\s+failed\\s+please$"
-    
```

Comment:
ok, we can remove this as it requires some more setup for startup, but maybe add additional file format-example.ci.yaml that will have this included?